docs/admin/kubelet.md

```diff
@@ -77,12 +77,12 @@ kubelet
       --cluster-domain="": Domain for this cluster.  If set, kubelet will configure all containers to search this domain in addition to the host's search domains
       --config="": Path to the config file or directory of files
       --configure-cbr0[=false]: If true, kubelet will configure cbr0 based on Node.Spec.PodCIDR.
-      --configure-hairpin-mode[=true]: If true, kubelet will set the hairpin mode flag on container interfaces. This allows endpoints of a Service to loadbalance back to themselves if they should try to access their own Service.
-      --container-runtime="docker": The container runtime to use. Possible values: 'docker', 'rkt'. Default: 'docker'.
+      --configure-hairpin-mode[=false]: If true, kubelet will set the hairpin mode flag on container interfaces. This allows endpoints of a Service to loadbalance back to themselves if they should try to access their own Service.
+      --container-runtime="": The container runtime to use. Possible values: 'docker', 'rkt'. Default: 'docker'.
```

Comment:
This is the one that stands out as confusing me, so maybe this needs regenerating?

I see the defaults.go sets the default to "docker" down below, but it appears to be lost here...